src/app/shared/post-it-collections/post-it-collections.component.ts

```diff
@@ -321,20 +332,28 @@ export class PostItCollectionsComponent implements OnInit, OnDestroy {
   }
 
   public emptyQuery(): boolean {
-    return Object.values(this.query).find(value => {
-      if (value.constructor === Array) {
-        value = value.length;
-      }
+    const queryValues = Object.values(this.query);
+    const notEmptyValue = queryValues.find(this.isNotEmpty);
+    return notEmptyValue === undefined;
```

Comment:
What exactly is being done here?

Reply:
Figuring out if query items are empty so that the component allows creating new collections if there is nothing searched.